### DISCUSSION TITLE
Quorum high latency

### DISCUSSION THREAD

**Discussion by eranshiffman95** [rabbitmq/rabbitmq-server#6834] [Questions] [answered]
Hello,
We are using rabbitMQ kubernetes cluster
https://github.com/rabbitmq/cluster-operator

I checked our services and we saw on Grafana that the delay is 8 times more than classic queue.
We don't want to lose the queue and the binding but we don't care about losing the messages when a pod is going down.

Is there any way that we can lower the latency?

**Comment by mkuratczyk** [comment, marked as answer]
You need to be more specific. If you are comparing a non-replicated classic queue to a replicated quorum queue, then the higher latency is pretty much inevitable. However, you didn't say what it is so it's impossible for us to asses whether you are observing a normal latency or some kind of an issue.

Please run  https://perftest.rabbitmq.com/ against your cluster to simulate what you are doing and share the results with us.

Having said that, if you don't need message replication then using a replicated quorum queue is unnecessary overhead.
